tests/test_api/test_validation.py

```diff
@@ -483,3 +483,47 @@ def test_validate_jobs():
     ]
     with raises(validation.GranuleValidationError):
         validation.validate_jobs(jobs)
+
+
+def test_check_bounds_formatting():
+    valid_jobs = [
+        {'job_parameters': {"bounds": [-10, 0, 10, 10]}},
+        {'job_parameters': {"bounds": [-180, -90, -170, -80]}},
+        {'job_parameters': {"bounds": [170, 75, 180, 90]}},
+    ]
+    invalid_jobs_valid_values = [
```

Comment:
I was a bit confused by the `invalid_jobs_valid_values` name, let's call it something like `invalid_jobs_bad_order`.

And for `invalid_jobs_invalid_values`, maybe something like `invalid_jobs_bad_values`.